### ISSUE TITLE
rat 

### ISSUE THREAD

**Issue by NukinGang** [Nyxonn/Discord-AIO#25]
rat doesnt work

**Comment by minhlongdev** [comment]
![image](https://user-images.githubusercontent.com/94881672/172039877-1fe3a7c1-2e78-4284-8d08-dd41930842ae.png)

**Comment by DuyLvzy123** [comment]
> ![image](https://user-images.githubusercontent.com/94881672/172039877-1fe3a7c1-2e78-4284-8d08-dd41930842ae.png)

please delete discord aio as fast as you can, there is bitcoin miner virus and if you used it I'm sorry.

**Comment by shawnlunsford** [comment]
> > ![image](https://user-images.githubusercontent.com/94881672/172039877-1fe3a7c1-2e78-4284-8d08-dd41930842ae.png)
> 
> please delete discord aio as fast as you can, there is bitcoin miner virus and if you used it I'm sorry.

![image](https://user-images.githubusercontent.com/14898342/172504454-06d4b2d9-0333-4bfc-b525-2ca431015769.png)
??

**Comment by Nyxonn** [comment]
Yes, there is a files to create monero miner as you can see in source code. I love when skid put open-source file into virus total and is spreading false informations about it..
